zoo-css-challenge/index.html

```diff
@@ -30,40 +32,42 @@ <h1>Birmingham Zoo</h1>
 		<section class="bears">
 			<h2>Bears</h2>
 			<p>Did you know that a bear's closest relative is the seal? Or that bears can smell over 2,000 times better than humans?</p>
-			<div class="image-container">
+			<div class="image-container" style="display:flex; justify-content:center">
 				<img
 					src="https://placebear.com/200/300"
 					alt="Grizzly Bear"
 					width="200"
 					height="300"
+					style="border: 10px solid teal;"
 				>
 				<img src="https://placebear.com/400/300"
 					alt="Grizzly Bear"
 					width="400"
 					height="300"
+					style="border: 10px solid rgb(2, 0, 128);"
 				>
 				<img src="https://placebear.com/200/400"
 					alt="Grizzly Bear"
 					width="200"
 					height="400"
+					style="border: 10px solid rgb(128, 0, 100);"
 				>
 			</div>
 		</section>
 
-		<section class="tigers">
-			<h2>Tiger</h2>
-			<h3>Top 5 Tiger Facts</h3>
-			<ol>
+		<section class="class">
+			<ul>
+			<h1 style="color:#FFFFFF">Tiger Facts</h1>
```

Comment:
```suggestion
			<h2>Tiger Facts</h2>
				<ul>
```
It looks like the heading and the list swapped places — we should swap them back so that `h2` comes first.

There's no task to change the heading level so we should change the `h1` back to a `h2`

The CSS styling should go in `style.css` file instead of the `style` attribute here